wp-admin/js/widgets/media-image-widget.js

```diff
@@ -147,11 +142,17 @@
 			 * customizer.
 			 */
 			mediaFrameContentView = mediaFrame.views.get( '.media-frame-content' )[0];
-			mediaFrameContentView.model.dfd.done( function() {
-				_.defer( function() { // Next tick.
-					mediaFrameContentView.$el.find( '[data-setting="caption"]:first' ).focus();
+			if ( mediaFrameContentView.model.dfd ) {
+				mediaFrameContentView.model.dfd.done( function() {
+					_.defer( function() { // Next tick.
+						mediaFrameContentView.$el.find( '[data-setting="caption"]:first' ).focus();
+					} );
 				} );
-			} );
+			} else {
+				_.defer( function() {
+					jQuery( '.media-frame-content [data-setting="caption"]:first' ).focus();
+				} );
+			}
```

Comment:
I'm happy to remove this logic here and agree the media modal should be taking care of the focus. It does this already focusing on the modal itself, which may be enough.